extensions/wikia/CreateNewWiki/tasks/SetCustomSettings.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Wikia\CreateNewWiki\Tasks;
+
+use User;
+use WikiFactory;
+use Wikia\Logger\Loggable;
+
+class SetCustomSettings implements Task {
+	use Loggable;
+
+	private $taskContext;
+
+	public function __construct( TaskContext $taskContext ) {
+		$this->taskContext = $taskContext;
+	}
+
+	public function prepare() {
+		return TaskResult::createForSuccess();
+	}
+
+	public function check() {
+		return TaskResult::createForSuccess();
+	}
+
+	public function run() {
+		global $wgUniversalCreationVariables, $wgLangCreationVariables;
+
+		$wikiType = "default";
```

Comment:
Is there some place to define const instead of hardcoding staff like "default", "univeral" etc? Maybe it's justified here, I don't know the full context but looks a bit like some fixed values